### PR TITLE
Get utils to pass pydocstyle

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
Fix docs on utils so pydocstyle passes.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Fix linting errors
- Enable pydocstyle to run in makefile so it gets run on CI

### Fixes 
- n/a

### CODE REVIEW

File: _delphi_utils_python/delphi_utils/archive.py [hunk 1]
```diff
@@ -99,7 +100,7 @@ def run_module(archive_type: str,
                cache_dir: str,
                export_dir: str,
                **kwargs):
-    """Builds and runs an ArchiveDiffer.
+    """Build and runs an ArchiveDiffer.
```
Comment (krivard) [inline _delphi_utils_python/delphi_utils/archive.py:103]:
```suggestion
    """Build and run an ArchiveDiffer.
```

File: _delphi_utils_python/delphi_utils/archive.py [hunk 2]
```diff
@@ -269,7 +273,7 @@ def filter_exports(self, common_diffs: FileDiffMap):
                 replace(diff_file, exported_file)
 
     def run(self):
-        """Runs the differ and archives the changed and new files."""
+        """Run the differ and archives the changed and new files."""
```
Comment (krivard) [inline _delphi_utils_python/delphi_utils/archive.py:276]:
```suggestion
        """Run the differ and archive the changed and new files."""
```

File: _delphi_utils_python/delphi_utils/archive.py [hunk 3]
```diff
@@ -330,9 +336,7 @@ def __init__(
         self.indicator_prefix = indicator_prefix
 
     def update_cache(self):
-        """
-        For making sure cache_dir is updated with all latest files from the S3 bucket.
-        """
+        """For making sure cache_dir is updated with all latest files from the S3 bucket."""
```
Comment (krivard) [inline _delphi_utils_python/delphi_utils/archive.py:339]:
```suggestion
        """Make sure cache_dir is updated with all latest files from the S3 bucket."""
```

File: _delphi_utils_python/delphi_utils/archive.py [hunk 4]
```diff
@@ -482,8 +490,9 @@ def archiving_branch(self):
 
     def update_cache(self):
         """
+        Check if cache_dir is clean: has everything nice committed if override_dirty=False.
```
Comment (krivard) [inline _delphi_utils_python/delphi_utils/archive.py:493]:
```suggestion
        Check if cache_dir is clean: has everything nicely committed if override_dirty=False.
```

**Review by krivard — CHANGES_REQUESTED** [review]
compound verb phrases like these probably aren't caught by the linter but we should be consistent

**Comment by chinandrew** [comment]
> compound verb phrases like these probably aren't caught by the linter but we should be consistent

Good catch, thanks

**Review by krivard — APPROVED** [review]
lgtm

**Pull request merged**